### ISSUE TITLE
NNs in this repo have ReLU activation functions

### ISSUE THREAD

**Issue by Robbybp** [SamChevalier/LoadShedVerification#1]
According to the paper, the NNs used should have sigmoid activation functions, but the 118-bus NNs in this repository have ReLU activation functions. For example:
```python
import torch
import os
fname = os.path.join("src", "outputs", "118_bus", "118_bus_128node.pt")
nn = torch.load(fname, weights_only=False)
print(nn)
```
```console
Sequential(
  (0): Linear(in_features=423, out_features=128, bias=True)
  (1): ReLU()
  (2): Linear(in_features=128, out_features=128, bias=True)
  (3): ReLU()
  (4): Linear(in_features=128, out_features=128, bias=True)
  (5): ReLU()
  (6): Linear(in_features=128, out_features=186, bias=True)
)
```
Did you upload the right ones?
